### PR TITLE
Cu 86bygzkm5 images placement for interoperability proxy identity

### PR DESCRIPTION
**Pull request by brett-onions**
## Summary by CodeRabbit

- **New Features**
  - Introduced environment variables for more dynamic configuration of OpenHIM services.
  - Added new environment variables for images, placements, instances, and replicas for various components.

- **Enhancements**
  - Updated image references for OpenHIM Console and MongoDB.
  - Adjusted CPU and memory limits for better performance.
  - Improved Kafka configuration.

- **Removals**
  - Removed outdated environment variable related to OpenHIM Console version.

### CODE REVIEW

**Comment by rcrichton** [comment]
Task linked: [CU-86bygzkm5 Images Placement for Interoperability, proxy, identity](https://app.clickup.com/t/86bygzkm5)

**Review by drizzentic — APPROVED** [review]
LGTM!

File: interoperability-layer-openhim/package-metadata.json [hunk 1]
```diff
@@ -6,9 +6,18 @@
   "type": "infrastructure",
   "dependencies": [],
   "environmentVariables": {
+    "OPENHIM_CORE_IMAGE": "jembi/openhim-core:v8.4.0",
+    "OPENHIM_CONSOLE_IMAGE": "jembi/openhim-console:v1.19.0",
```
Comment (drizzentic) [inline interoperability-layer-openhim/package-metadata.json:10]:
```suggestion
    "OPENHIM_CONSOLE_IMAGE": "openhim-console:v1.18.2",
```

File: interoperability-layer-openhim/docker-compose.yml [hunk 2]
```diff
@@ -39,7 +39,7 @@ services:
         - prometheus-port=8080
 
   openhim-console:
-    image: ${OPENHIM_CONSOLE_VERSION}
+    image: ${OPENHIM_CONSOLE_IMAGE}
```
Comment (BMartinos) [inline interoperability-layer-openhim/docker-compose.yml:42]: I think there is already a env for defining the tag to use, but think the updated env naming is more in line with the others. We should then remove the older env:
https://github.com/jembi/platform/pull/289/files#diff-d1dbb584ce75d5c7c1a32820718b2e21ce4fa3fcce44cb1a473ba38a527293c2R38

File: interoperability-layer-openhim/package-metadata.json [hunk 3]
```diff
@@ -7,9 +7,17 @@
   "dependencies": [],
   "environmentVariables": {
     "OPENHIM_CORE_IMAGE": "jembi/openhim-core:microfrontends-pre-release",
+    "OPENHIM_CONSOLE_IMAGE": "jembi/openhim-console:v1.18.2",
```
Comment (BMartinos) [inline interoperability-layer-openhim/package-metadata.json:10]: Im wondering if this image tag for console shouldnt be pointing the the new microfrontends one?

**Review by BMartinos — COMMENTED** [review]
@brett-onions overall looks fine to me, just two comments around the existing console env for the image/version to use

**Review by BMartinos — APPROVED** [review]

**Pull request merged**